src/test/storage/storage_manager_test.cpp

```diff
@@ -50,4 +50,24 @@ TEST_F(StorageStorageManagerTest, HasTable) {
   EXPECT_EQ(storage_manager.has_table("first_table"), true);
 }
 
+TEST_F(StorageStorageManagerTest, TableNames) {
+  auto& storage_manager = StorageManager::get();
+  auto expected_tables = std::vector<std::string>{"first_table", "second_table"};
+  // the return vector is supposed to be sorted, so sort this expected values vector as well to ensure consistency
```

Comment:
We recommend to sort here or not sort at all because the order of elements is deterministic using a hash function of the map.

Reply:
> We recommend to sort here or not sort at all because the order of elements is deterministic using a hash function of the map.

But per default, the unordered hash map uses `std::hash`, which is implementation-dependent.